core/shark_turbine/dynamo/executor.py

```diff
@@ -153,7 +153,7 @@ def __init__(
         self,
         user_module: VmModule,
         device_state: DeviceState,
-        entry_name: str = "main",
+        entry_name: str = "main$async",
```

Comment:
Python noob Q: what does the `$` sign do? is this doing something similar to `f"main{async}"` ?

Reply:
Nothing. IREE always exports an "$async" suffixed function when compiling from torch. This is selecting that variant.